GOVERANCE.md

```diff
@@ -0,0 +1,200 @@
+renewed by the committee annually.
+
+Should a member of the Maintainer Committee cease being active in the Subproject,
+violate the Code Of Conduct, or need to be removed for some other reason, they
+may be removed by a 2/3 majority vote of the other Committee members, or a
+majority vote of the Steering Committee.
+
+## Adding New Subprojects
+
+During the monthly Steering meeting, any project member may recommend projects
+to become part of Open Cluster Management. These projects should have the following
+characteristics:
+
+- Have a mission of Kubernetes cluster and multicluster management;
+- Are appropriately licensed and governed or willing to become so;
+- Are under active development;
+- Consist of high quality code and designs.
+
+Before submitting an application to the Steering Committee, the applying project
+must hold an internal consensus vote of all major contributors to join
+Open Cluster Management. The Steering Committee will then review the
+application, and decide whether or not to accept it. If it is accepted, the Committee
+will assign one person to assist the Subproject in their integration.
+
+In some cases, promising but incomplete projects may be accepted as Experimental
+Subprojects. Such Experimental Subprojects will be considered part of
+Open Cluster Management, but will be marked as Experimental on the website and in code
+repositories, in order to inform users. Experimental Subproject Members are considered
+Members of Open Cluster Management, but the Subproject is not entitled to a representative on the
+Steering Committee. Steering will review Experimental Subprojects at least twice
+per year to determine if they have matured to full Subproject status.
+
+New Subprojects will be initialized with a set of maintainers, reviewers,
+contributors, and other roles assigned based on the people doing the work before
+the Subproject was accepted. The appropriate contributors will be added as
+Organization Members without the regular process.
+
+## Removing Projects
+
+In some cases, projects will become inactive or unmaintainable, or wish to separate
+from Open Cluster Management. Any Steering Committee member may propose removal of a project on
+these grounds, and Steering can confirm this with a majority vote. The CNCF will
+determine which, if any, assets of those subprojects will be removable should a
+project leave OCM.
+
+Where possible, Projects which still have contributors will then be moved to a
+repository in their own namespace. Projects which have ceased all activity are
+moved to an archive namespace.
+
+# Steering Committee
+
+The overall Open Cluster Management umbrella project is governed by a Steering Committee(SC).
+
+## Duties
+
+The Steering Committee is responsible for the following tasks, any of which may
+be delegated to a person or group selected by the committee:
+
+- Reviewing and deciding on new projects to add to Open Cluster Management
```

Comment:
```suggestion
- Reviewing and deciding on new projects to add to Open Cluster Management.
```

Reply:
Same comment for the rest of the bullet points.

Reply:
Updated.